backend/script/test

```diff
@@ -8,5 +8,5 @@ poetry run black --check app
 poetry run mypy app
 poetry run ruff check app
 
-mkdir -p ../test-results
+mkdir -p "$(dirname "$0")/../.."/test-results
```

Comment:
`cd` into the directory so the next line still works regardless of cwd.

Reply:
I had trouble `cd`ing into the directory without GitHub Actions throwing an error.
So I kept `test-results` in /backend and changed the `test-results` path in the CI workflow.